experimental/QuadFormAndIsom/src/enumeration.jl

```diff
@@ -461,7 +465,7 @@ function representatives_of_hermitian_type(Lf::ZZLatWithIsom, m::Int = 1)
   ok, rk = divides(rk, euler_phi(n*m))
   ok || return reps
 
-  gene = Hecke.HermGenus[]
+  gene = HermGenus[]
```

Comment:
`HermGenus` is not a concrete type if I remember correctly. There should be a `hermitian_genus_type(E)` or something like that in Hecke.

Reply:
Ah yes, indeed, thanks! I forgot about that function..